### ISSUE TITLE
Feature request: Live Recording transcribing

### ISSUE THREAD

**Issue by nomatica** [Elmari/easytranscript#2]
Record speakers while transcribing. 

I am pretty sure this is out of the realm of possibility here because I think it is outside the scope . But I often have the opportunity to listen to the presentation live that I will to transcribe.  It would be helpful to take initial notes while doing so. I am looking for a transcription application that allows me to record, and annotate simultaneously..

**Comment by Elmari** [comment]
Good Idea! Currently this is not possible but i'll add it to the feature wishlist for version 3.
